### PR TITLE
Update running instructions

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ After opening the project in IntelliJ, double check that the Java SDK is properl
 Presto comes with sample configuration that should work out-of-the-box for development. Use the following options to create a run configuration:
 
 * Main Class: `io.prestosql.server.PrestoServer`
-* VM Options: `-ea -XX:+UseG1GC -XX:G1HeapRegionSize=32M -XX:+UseGCOverheadLimit -XX:+ExplicitGCInvokesConcurrent -Xmx2G -Dconfig=etc/config.properties -Dlog.levels-file=etc/log.properties -Djdk.attach.allowAttachSelf=true`
+* VM Options: `-ea -XX:+UseG1GC -XX:G1HeapRegionSize=32M -XX:+UseGCOverheadLimit -XX:+ExplicitGCInvokesConcurrent -Xmx2G -Dconfig=etc/config.properties -Dlog.levels-file=etc/log.properties -Djdk.attach.allowAttachSelf=true -Dpresto-temporarily-allow-java8=true`
```
Comment (losipiuk) [inline README.md:42]: Shouldn't we change the paragraph above to state that one should use JDK 11 instead? (language level may still stay at 8.0)

Comment (wendigo) [reply README.md:42]: to be honest, idk, let's ask @findepi

Comment (martint) [reply README.md:42]: Yes, this should talk about running with Java 11. We're going to make that a requirement soon.

Comment (wendigo) [reply README.md:42]: @martint @losipiuk take a look now

File: README.md [hunk 2]
```diff
@@ -49,6 +49,8 @@ Additionally, the Hive plugin must be configured with the location of your Hive
 
     -Dhive.metastore.uri=thrift://localhost:9083
 
+When running on JDK8 `VM Options` must be supplemented with `-Dpresto-temporarily-allow-java8=true` in order to Presto to start.
```
Comment (findepi) [inline README.md:52]:
```suggestion
When running on Java 8 `VM Options` must be supplemented with `-Dpresto-temporarily-allow-java8=true` in order to Presto to start.    
```

File: README.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ See the [User Manual](https://prestosql.io/docs/current/) for deployment instruc
 ## Requirements
 
 * Mac OS X or Linux
-* Java 8 Update 161 or higher (8u161+), 64-bit. Both Oracle JDK and OpenJDK are supported.
+* Java 8 Update 161 or higher (8u161+), 64-bit is minimal supported version. Java 11 is recommended although it's not required yet. Both Oracle JDK and OpenJDK are supported.
```
Comment (findepi) [inline README.md:12]:
```suggestion
* Java 8 Update 161 or higher (8u161+), 64-bit is minimal supported version. Java 11 is recommended and will
  soon become required.
```

File: README.md [hunk 4]
```diff
@@ -49,6 +49,8 @@ Additionally, the Hive plugin must be configured with the location of your Hive
 
     -Dhive.metastore.uri=thrift://localhost:9083
 
+When running on Java 8 `VM Options` must be supplemented with `-Dpresto-temporarily-allow-java8=true` in order to Presto to start.
```
Comment (findepi) [inline README.md:52]:
```suggestion
When running on Java 8, the VM Options must be supplemented with `-Dpresto-temporarily-allow-java8=true` in order for Presto to start.
```

File: README.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ See the [User Manual](https://prestosql.io/docs/current/) for deployment instruc
 ## Requirements
 
 * Mac OS X or Linux
-* Java 8 Update 161 or higher (8u161+), 64-bit. Both Oracle JDK and OpenJDK are supported.
+* Java 8 Update 161 or higher (8u161+), 64-bit is minimal supported version. Java 11 is recommended and will soon become required. Both Oracle JDK and OpenJDK are supported.
```
Comment (martint) [inline README.md:12]: *is the minimum supported ...*

**Pull request merged**